usso/jwt_test.go

```diff
@@ -0,0 +1,100 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017-2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package usso
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/juju/usso/openid"
+)
+
+type testJWT struct {
+	resp     openid.Response
+	expected []string
+}
+
+func TestNewJWTToken(t *testing.T) {
+	test1 := testJWT{
+		resp:     openid.Response{ID: "id", Teams: []string{"teamone", "team2"}},
+		expected: []string{"", "", ""},
+	}
+	test2 := testJWT{
+		resp:     openid.Response{ID: "id", Teams: []string{"teamone", "team2"}, SReg: map[string]string{"nickname": "jwt"}},
+		expected: []string{"jwt", "", ""},
+	}
+	test3 := testJWT{
+		resp:     openid.Response{ID: "id", Teams: []string{"teamone", "team2"}, SReg: map[string]string{"nickname": "jwt", "email": "jwt@example.com", "fullname": "John W Thompson"}},
```

Comment:
just a matter of format: these lines are too long. Can you break the line like resp:     
                openid.Response{
			ID: "id", 
			Teams: []string{"teamone", "team2"}, 
			SReg: map[string]string{"nickname": "jwt", "email": "jwt@example.com", "fullname": "John W Thompson"}},
                }